include/rppt_tensor_audio_augmentations.h

```diff
@@ -36,6 +36,9 @@ SOFTWARE.
  * @{
  */
 
+/*! \brief RPP maximum channels in audio tensor \ingroup group_rppdefs \page subpage_rppt */
+#define RPPT_MAX_CHANNELS   ( 16 )
```

Comment:
Is there another RPPT_MAX_CHANNELS that we use elsewhere? If not, please move to rppdefs.h and change name:
```
/*! \brief RPP maximum dimensions in tensor \ingroup group_rppdefs \page subpage_rppt */
#define RPPT_MAX_DIMS   ( 5 )
/*! \brief RPP maximum channels in audio tensor \ingroup group_rppdefs \page subpage_rppt */
#define RPPT_AUDIO_MAX_CHANNELS   ( 16 )
```